app-crypt/gcr/gcr-3.92.0.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
```

Comment:
We can drop this \o/

The `pkgdiff -b` output contains

````diff
-# Post-install scripts
-meson.add_install_script('meson_post_install.py',
-  get_option('datadir'),
-  get_option('libdir'),
-  gcr_major_version.to_string(),
-  gcr_soversion,
-)
````

which was the only thing using Python during the build. To reproduce and see the failure in v3.41.1 you have to unmask the `native-symlinks` flag on `dev-lang/python-exec` and disable it.